provider/cluster/inventory.go

```diff
@@ -320,40 +317,32 @@ func (is *inventoryService) runCheck(ctx context.Context) <-chan runner.Result {
 	})
 }
 
-func (is *inventoryService) getStatus(
-	inventory []Node, reservations []*reservation) InventoryStatus {
-
-	status := InventoryStatus{}
+func (is *inventoryService) getStatus(inventory []Node, reservations []*reservation) (status InventoryStatus) {
```

Comment:
:nail_care: let's only use the `(x Foo)` when it's absolutely necessary (catching a `panic`).